src/cell_gater/widgets/scatter_widget.py

```diff
@@ -29,7 +30,7 @@ class ScatterInputWidget(QWidget):
     def __init__(self, model: DataModel, viewer: Viewer) -> None:
         super().__init__()
 
-        self.setLayout(QVBoxLayout())
+        self.setLayout(QGridLayout())
```

Comment:
doing this makes the overall reserved space for all widgets displayed on the right larger when you are not properly dealing with sizes. What you already can see now is that the left column for loading regionprops csvs takes up a lot of extra space due to this change. If you want this you need to ensure that minimum width is respected.

Reply:
alright, I would then take a look at this UI things after the scatter plot widget is functional